src/style/product_card.css

```diff
@@ -0,0 +1,109 @@
+:root {
+  --color-text-fat: #060b35;
+  --color-text-small: #616070;
+  --color-accent: #00acdc;
+}
+
+.card__price,
+.card__reviews,
+.card__product-code {
+  font-family: "Roboto-Regular", sans-serif;
```

Comment:
no need to use different `fon-family`. just change `font-weigth`. also, it would be easier to use google fonts and set in `link` tag